pkg/kontainer-engine/drivers/util/utils.go

```diff
@@ -9,6 +9,7 @@ import (
 	v3 "github.com/rancher/rke/types"
 	"gopkg.in/yaml.v2"
 	v1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
```

Comment:
Do we need to keep both rbac/v1beta1 and rbac/v1 to support multiple k8s versions?

Reply:
I could be missing something, but I see both in 1.18 (the lowest version we need to support) - 

v1: https://github.com/kubernetes/kubernetes/blob/release-1.18/staging/src/k8s.io/api/rbac/v1/types.go 
v1beta1: https://github.com/kubernetes/kubernetes/blob/release-1.18/staging/src/k8s.io/api/rbac/v1beta1/types.go

Reply:
JFYI here is a table I've compiled a while ago with old and replacement APIs and k8s versions they are available starting from ([internal SUSE link](https://confluence.suse.com/display/EN/Kubernetes+1.22+Checklist#Kubernetes1.22Checklist-k8s-1.22-removals)).

Reply:
Both is supported, but I wonder if we can just update everything to v1 if there is nothing tied to v1beta1 strictly? Or maybe I am missing something.

https://github.com/rancher/rancher/blob/f2b697bbc860213fc7720aa106fc6b0b6ecc33d2/pkg/kontainer-engine/drivers/util/utils.go#L129-L131
https://github.com/rancher/rancher/blob/f2b697bbc860213fc7720aa106fc6b0b6ecc33d2/pkg/kontainer-engine/drivers/util/utils.go#L90

Reply:
@prachidamle , ah, misunderstood your original comment. Yes, I think we want to update those occurrences you found to v1 too and get rid of references to `k8s.io/api/rbac/v1beta1` altogether.

Reply:
Oh! yep I misunderstood too, I thought the question was if updating it to all to `v1` was safe. Will update everything to rbac v1

Reply:
cool, thanks